scipy/integrate/_quadrature.py

```diff
@@ -1035,3 +1038,235 @@ def newton_cotes(rn, equal=0):
+        will be initialized with the number of dimensions determine from
+        `ranges`.
+    log : boolean, default: False
+        When set to True, `func` returns the log of the integrand, and
+        the result object contains the log of the integral.
+    args : sequence, optional
+        Extra arguments to pass to `func`. All extra arguments must be packed
+        into a sequence; the will be unpacked to be passed to `func`, e.g.
+        ``func(x1, x2, *args)``.
+
+    Returns
+    -------
+    result : object
+        A result object with attributes:
+
+        integral : float
+            The estimate of the integral.
+        standard_error :
+            The error estimate. See Notes for interpretation.
+
+    Notes
+    -----
+    Evaluating the integrand at each of the `n_points` points in the QMC sample
+    generates an estimate of the integral. By applying `n_offsets` random
+    offsets to the QMC sample before calculating the integral in this way, we
+    draw `n_offsets` i.i.d. random samples from a population of integral
+    estimates. The sample mean :math:`m` of these integral estimates is an
+    unbiased estimator of the true value of the integral, and the standard
+    error of the mean :math:`s` of these estimates may be used to generate
+    confidence intervals using the t distribution with ``n_offsets - 1``
+    degrees of freedom. For details, see [1]_.
+
+    References
+    ----------
+    .. [1] Joe, Stephen. "Randomization of lattice rules for numerical multiple
+           integration." Journal of Computational and Applied Mathematics 31.2
+           (1990): 299-304.
+
+    Examples
+    --------
+    QMC quadrature is particularly useful for computing integrals in higher
+    dimensions; an example integrand is the probability density function
+    of a multivariate normal distribution.
+
+    >>> import numpy as np
+    >>> from scipy import stats
+    >>>
+    >>> dim = 8
+    >>> mean = np.zeros(dim)
+    >>> cov = np.eye(dim) / 2**8
+    >>>
+    >>> def func(*z):
+    ...     z = np.asarray(z).T
+    ...     return stats.multivariate_normal.pdf(z, mean, cov)
+
+    To compute the integral over the unit hypercube:
+
+    >>> from scipy.integrate import qmc_quad
+    >>> lb = np.zeros(dim)
+    >>> ub = np.ones(dim)
+    >>> ranges = np.asarray([lb, ub]).T
+    >>> rng = np.random.default_rng(1638083107694713882823079058616272161)
+    >>> qrng = stats.qmc.Halton(d=dim, seed=rng)
+    >>> n_points, n_offsets = 256, 8
+    >>> res = qmc_quad(func, ranges)
+    >>> res.integral, res.standard_error
+    (0.0001842819684617149, 1.3959190979748066e-07)
+
+    A two-sided, 99% confidence interval for the integral may be estimated
+    as:
+
+    >>> t = stats.t(df=n_offsets-1, loc=res.integral, scale=res.standard_error)
+    >>> t.interval(0.99)
+    (0.00018389017561108015, 0.00018461661169997918)
+
+    Indeed, the value reported by `scipy.stats.multivariate_normal.cdf` is
```

Comment:
```suggestion
    Indeed, the value reported by `scipy.stats.multivariate_normal` is
```
I think this is the documentation build issue.